eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -14,21 +14,28 @@ resources:
       env:
         ROOTFS_DIR: /crossrootfs/armv6
 
+    - container: mono_linux_arm64
+      image: cr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-cross-arm64
+      env:
+        ROOTFS_DIR: /crossrootfs/arm64
+
```

Comment:
Why are we using an ubuntu base for this particular target. Is this the particular case in our build where we have to care about the differences between build, host, and target platforms as all three differ?

Reply:
Can we put a mention of the issue number in the YAML?